Composer/packages/client/src/recoilModel/undo/history.ts

```diff
@@ -184,11 +163,11 @@ export const UndoRoot = React.memo((props: UndoRootProps) => {
   });
 
   const canUndo = () => {
-    return history.canUndo();
+    return history?.canUndo?.();
```

Comment:
It should not hit the component with an empty history object. Basically only when a project is loaded it should hit here